tests/integration/suite/conftest.py

```diff
@@ -101,7 +102,7 @@ def admin_mc():
     """Returns a ManagementContext for the default global admin user."""
     r = requests.post(AUTH_URL, json={
         'username': 'admin',
```

Comment:
not sure if it even matters, but since password is abstracted to `SERVER_PASSWORD`, would we want to do the same but for `SERVER_USERNAME`?

Reply:
It will always be admin, as this the main account.